src/main/java/frc/team166/robot/subsystems/LED.java

```diff
@@ -0,0 +1,339 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.team166.robot.subsystems;
+
+import javax.lang.model.util.ElementScanner6;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.Victor;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.team166.chopshoplib.commands.SubsystemCommand;
+
+public class LED extends Subsystem {
+    //these will be changed from Victors to something else when we get real hardware...
+    Victor blue = new Victor(1);
+    Victor green = new Victor(2);
+    Victor red = new Victor(3);
+
+    public LED() {
+        SmartDashboard.putData("green", greenOn());
+        SmartDashboard.putData("red", redOn());
+        SmartDashboard.putData("blue", blueOn());
+        SmartDashboard.putData("cyan", cyanOn());
+        SmartDashboard.putData("color cycle", colorCycle());
+        SmartDashboard.putData("light team color", lightTeamColor());
+        SmartDashboard.putData("flash team color", blinkTeamColor());
+        SmartDashboard.putData("pulse team color", pulseTeamColor());
+        SmartDashboard.putData("flash green", blinkGreen(9));
+    }
+
+    private void safeDelay(double secondsToWait) {
+        double lastUpdateTime = System.currentTimeMillis();
+        while (System.currentTimeMillis() < lastUpdateTime + (secondsToWait * 1000)) {
+        }
+    }
+
+    private boolean isBlueTeam() {
+        Alliance team = DriverStation.getInstance().getAlliance();
+        if (team == DriverStation.Alliance.Blue) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    private void setTeamColor(boolean turnOn) {
+        if (isBlueTeam()) {
+            red.set(0);
+            if (turnOn) {
+                blue.set(1);
+            } else {
+                blue.set(0);
+            }
+        } else {
+            blue.set(0);
+            if (turnOn) {
+                red.set(1);
+            } else {
+                red.set(0);
+            }
+        }
+    }
+
+    private boolean pulseColor(Victor color, double step, boolean goUp) {
+        if (color.get() >= 1) {
+            goUp = false;
+        }
+        if (color.get() <= 0) {
+            goUp = true;
+        }
+
```

Comment:
Add a comment here noting that we need to fix this later (something like //TODO: add delay here)